service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/ControlledAzureDatabaseResource.java

```diff
@@ -138,7 +143,28 @@ public void addCreateSteps(
       FlightBeanBag flightBeanBag) {
     RetryRule cloudRetry = RetryRules.cloud();
 
-    flight.addStep(
+    getAddSteps(flightBeanBag).forEach(step -> flight.addStep(step, cloudRetry));
```

Comment:
Github is not showing this change very well. I extracted creation of the list of the steps into a separate method so I could test it more easily.